### PR TITLE
feat: add full angular demo

### PR DESCRIPTION
**Pull request by steve-haar**

### CODE REVIEW

File: grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts [hunk 1]
```diff
@@ -23,11 +23,12 @@ export class CustomClrColumnOrderingColumnDirective {
   }
 
   @HostListener('keydown', ['$event']) keydown(event: KeyboardEvent) {
+    const isColumnTarget = (event.target as HTMLElement)?.tagName === 'CLR-DG-COLUMN';
```
Comment (kevinbuhmann) [inline grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts:26]: I think this should be in a separate commit that explains what the change does.

File: grid-demo/src/app/angular/final-demo/final-demo.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+@Component({
+  templateUrl: './final-demo.component.html',
+  styleUrls: ['./final-demo.component.scss'],
+  providers: [LazyLoadService],
+})
+export class AngularFinalDemoComponent {
+  readonly data: Observable<{ vms: Vm[]; totalResults: number; loadedCount: number }>;
+  columns = columns.map(column => ({ ...column, hidden: false }));
+  clrDgSelectable = true;
```
Comment (kevinbuhmann) [inline grid-demo/src/app/angular/final-demo/final-demo.component.ts:17]: I don't think this property is used.

File: grid-demo/src/app/angular/final-demo/final-demo.component.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  constructor(private readonly lazyLoadService: LazyLoadService, changeDetectorRef: ChangeDetectorRef) {
```
Comment (kevinbuhmann) [inline grid-demo/src/app/angular/final-demo/final-demo.component.ts:19]: `changeDetectorRef` should be marked `private readonly`.

File: grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts (hunk 1 above)
Comment (steve-haar) [reply grid-demo/src/app/angular/column-ordering/custom-clr-column-ordering-column.directive.ts:26]: fixed

File: grid-demo/src/app/angular/final-demo/final-demo.component.ts (hunk 2 above)
Comment (steve-haar) [reply grid-demo/src/app/angular/final-demo/final-demo.component.ts]: removed

File: grid-demo/src/app/angular/final-demo/final-demo.component.ts (hunk 3 above)
Comment (steve-haar) [reply grid-demo/src/app/angular/final-demo/final-demo.component.ts:18]: I don't need it to be a property though, it's just a constructor parameter

Comment (kevinbuhmann) [reply grid-demo/src/app/angular/final-demo/final-demo.component.ts:18]: Oh right, I am dumb. Don't mind me.

**Review by kevinbuhmann — APPROVED** [review]

**Comment by kevinbuhmann** [comment]
"chore: only grab column if focused on root column cell" sounds like a fix.
